### ISSUE TITLE
java.lang.NoSuchMethodError: com.intellij.notification.NotificationGroup.createNotification

### ISSUE THREAD

**Issue by ivanporty** [GoogleCloudPlatform/cloud-code-intellij#2771]
From cloud-code-intellij-internal created by [etanshaul](https://github.com/etanshaul): GoogleCloudPlatform/cloud-code-intellij-internal#1803  

From cloud-code-intellij created by [ivanporty](https://github.com/ivanporty): GoogleCloudPlatform/cloud-code-intellij#2741  

```
Stack trace:
java.lang.NoSuchMethodError: com.intellij.notification.NotificationGroup.createNotification$default(Lcom/intellij/notification/NotificationGroup;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Lcom/intellij/notification/NotificationType;Lcom/intellij/notification/NotificationListener;ILjava/lang/Object;)Lcom/intellij/notification/Notification;
	at com.google.cloud.tools.intellij.kubernetes.dependencies.ManagedDepsInstallerTask.onSuccess(ManagedDepsInstallerTask.kt:68)
	at com.google.cloud.tools.intellij.kubernetes.dependencies.InstallerTask.onSuccess(InstallerTask.kt:93)
	at com.intellij.openapi.progress.impl.CoreProgressManager.finishTask(CoreProgressManager.java:501)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.lambda$run$1(CoreProgressManager.java:420)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:309)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:441)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:424)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:407)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:908)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:781)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:424)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:698)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:423)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

app.name.version
PyCharm
app.version.major
2019
app.version.minor
3.4
java.version
1.8.0_232
java.vm.vendor
AdoptOpenJDK
os.name
Mac OS X
plugin.version
20.4.2-SNAPSHOT

**Comment by ivanporty** [comment]
```
java.lang.NoSuchMethodError: 'com.intellij.notification.Notification com.intellij.notification.NotificationGroup.createNotification$default(com.intellij.notification.NotificationGroup, java.lang.String, java.lang.String, java.lang.String, com.intellij.notification.NotificationType, com.intellij.notification.NotificationListener, int, java.lang.Object)'
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldConfigurationDetector.createNotification(SkaffoldConfigurationDetector.kt:175)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldConfigurationDetector.createNotification$default(SkaffoldConfigurationDetector.kt:173)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldConfigurationDetector.showPromptForSkaffoldConfigurations(SkaffoldConfigurationDetector.kt:142)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldConfigurationDetector$projectOpened$1.run(SkaffoldConfigurationDetector.kt:88)
```

**Comment by ivanporty** [comment]
Will be fixed in the next release.

**Comment by etanshaul** [comment]
fixed via `20.6.1`
